data_processing/geozones/task_functions.py

```diff
@@ -0,0 +1,264 @@
+        ((df['type'] == 'Arrondissement') & (df['nom'].str.contains('|'.join(['Paris', 'Lyon', 'Marseille']))))
+    ]
+    df = df.loc[
+        df['level'] != 'country'
+    ]
+
+    # get countries (with ISO alpha 2 code) from another source
+    countries = pd.read_csv(
+        "https://www.data.gouv.fr/fr/datasets/r/2b38f28d-15e7-4f0c-b61d-6ca1d9b1cfa2",
+        sep=';',
+        encoding='cp1252',
+        dtype=str
+    )
+    countries = countries.loc[countries['SOUVERAIN'] == 'O']
+    countries['uri'] = countries['CODE_COG'].apply(
+        lambda x: "http://id.insee.fr/geo/etat/" + x if isinstance(x, str) else x
+    )
+    countries.rename({
+        'NOM_LONG_ETAT': 'nom',
+        'NOM_COURT': 'nomSansArticle',
+    }, axis=1, inplace=True)
+    countries['codeArticle'] = None
+    countries['type'] = 'country'
+    countries['is_deleted'] = False
+    countries['level'] = 'country'
+    countries['codeINSEE'] = countries['ISO_alpha2'].apply(
+        lambda x: x.lower() if isinstance(x, str) else x
+    )
+    countries['_id'] = countries['ISO_alpha2'].apply(
+        lambda x: "country:" + x.lower() if isinstance(x, str) else x
+    )
+    countries = countries[
+        ['uri', 'nom', 'codeINSEE', 'nomSansArticle', 'codeArticle', 'type', 'is_deleted', 'level', '_id']
+    ]
+    countries_json = json.loads(countries.to_json(orient='records'))
+
+    export = json.loads(df.to_json(orient='records'))
+    export.extend([
+        {
+            "uri": "http://id.insee.fr/geo/world",
+            "nom": "Monde",
+            "codeINSEE": "world",
+            "nomSansArticle": "Monde",
+            "codeArticle": None,
+            "type": "country-group",
+            "is_deleted": False,
+            "level": "country-group",
+            "_id": "country-group:world"
+        },
+        {
+            "uri": "http://id.insee.fr/geo/europe",
+            "nom": "Union Européenne",
+            "codeINSEE": "ue",
+            "nomSansArticle": "Union Européenne",
+            "codeArticle": None,
+            "type": "country-group",
+            "is_deleted": False,
+            "level": "country-group",
+            "_id": "country-group:ue"
+        },
+        {
+            "uri": None,
+            "nom": "DROM",
+            "codeINSEE": "fr:drom",
+            "nomSansArticle": "DROM",
+            "codeArticle": None,
+            "type": "country-subset",
+            "is_deleted": False,
+            "level": "country-subset",
+            "_id": "country-subset:fr:drom"
+        },
+        {
+            "uri": None,
+            "nom": "DROM-COM",
+            "codeINSEE": "fr:dromcom",
+            "nomSansArticle": "DROM-COM",
+            "codeArticle": None,
+            "type": "country-subset",
+            "is_deleted": False,
+            "level": "country-subset",
+            "_id": "country-subset:fr:dromcom"
+        },
+        {
+            "uri": None,
+            "nom": "Metropolitan France",
```

Comment:
I suggest to use the French name:
```suggestion
            "nom": "France métropolitaine",
```